src/UI/Component/Button/Factory.php

```diff
@@ -260,4 +260,38 @@ public function tag($label, $action);
 	 */
 	public function bulky($icon_or_glyph, $label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Toggle Button triggers the activation/deactivation of some content already shown on the screen, i.e.
+	 *       a filter. The deactivation of content means, that it is still shown and the user can still interact
+	 *       with it, but it has no effect to the system.
```

Comment:
So we do not want checkbox like data input? E.g. to witch on and off permissions, correct? Toggling things in tables, right? Maybe list the checkbox as rival then and describe the difference in a sentence there., to make this really clear.

Reply:
I would very much appreciate to have Toggles in forms as well. To me they are slightly different to checkboxes. While the former imho convey the meaning of "turning on some functionality" the latter to me seem to be about "confirming some statement". This implies that a lot of places in ILIAS could benefit of toggles (think of the whole "Additional Features"-shebang).

Reply:
I added Checkboxes to the rivals to clarify the difference between Checkboxes and Toggle Buttons.

Reply:
* We suggest to replace "content" with "controls" here.
* ... "effect _on_ the system"